srcpkgs/fstrcmp/template

```diff
@@ -0,0 +1,43 @@
+# Template file for 'fstrcmp'
+pkgname=fstrcmp
+version=0.7.D001
+revision=1
+build_style=gnu-configure
+configure_args="LIBTOOL=${XBPS_CROSS_BASE}/usr/bin/libtool"
+make_build_args="LIBTOOL=${XBPS_CROSS_BASE}/usr/bin/libtool"
+make_install_args="LIBTOOL=${XBPS_CROSS_BASE}/usr/bin/libtool"
+hostmakedepends="automake groff ghostscript"
+makedepends="libtool"
+short_desc="Fuzzy comparison of strings"
+maintainer="Elie ROUDNINSKI <xademax@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="http://${pkgname}.sourceforge.net/"
+distfiles="http://${pkgname}.sourceforge.net/${pkgname}-${version}.tar.gz"
+checksum=e4018e850f80700acee8da296e56e15b1eef711ab15157e542e7d7e1237c3476
+
+fstrcmp-doc_package() {
+	short_desc+=" - documentation files"
+	pkg_install() {
+		vmove usr/share/doc
+	}
+}
+
+libfstrcmp_package() {
+	short_desc+=" - runtime library"
+	pkg_install() {
+		vmove "usr/lib/libfstrcmp.so.*"
+	}
+}
+
+libfstrcmp-devel_package() {
+	short_desc+=" - development files"
+	depends="libfstrcmp>=${version}_${revision}"
+	pkg_install() {
+		vmove usr/include
+		vmove usr/lib/pkgconfig
+		vmove "usr/lib/*.so"
+		vmove usr/lib/libfstrcmp.a
+		vmove usr/lib/libfstrcmp.la
```

Comment:
Do we really need libtool archives for this package ?

Reply:
Not really my call here. I saw other packages keeping their .la so did I.

Reply:
Ok, i will remove it then :+1: